warp10/build.gradle

```diff
@@ -40,10 +40,10 @@ dependencies {
     api 'org.bouncycastle:bcpg-jdk15on:1.70'
 
     // https://mvnrepository.com/artifact/org.foundationdb/fdb-java
-    api 'org.foundationdb:fdb-java:7.1.25'
+    api 'org.foundationdb:fdb-java:7.2.5'
```

Comment:
It fails with fdb 7.1.26 installed. 
```
Caught exception when initializing FoundationDB API version, please ensure the FoundationDB clients package is installed.
```

Need to fix FDBUtils error message to add the exeption detail in logs and runtime exception, such as:
```
FDBException: This version of the FoundationDB Java binding is not supported by the installed FoundationDB C library. The binding requires a library that supports API version 720, but the installed library supports a maximum version of 710.
```
With latest release installed (7.1.30), it also fails with DEFAULT_FDB_API_VERSION 710.

With latest release installed (7.1.30), it also fails with DEFAULT_FDB_API_VERSION 720.
It also fails with fdb-java:7.2.0

Better to update to fdb-java:7.1.29 , and keep protocol version 710, it seems to early for 7.2.x and 720.

Reply:
My mistake, there is a 7.2.4 release (without avx). This one works with fdb-java:7.2.5, and protocol 710 and 720.

Following the previous comment, there is no point to keep api version to 710, because fdb-java:7.2.5 force the installation of fdb 7.2.4

So DEFAULT_FDB_API_VERSION can be updated, and 10-fdb.conf.template should be updated and line should be commented, as this is the default.